powermon/commands/trigger.py

```diff
@@ -17,52 +18,52 @@ class TriggerType(LowercaseStrEnum):
 
 class Trigger:
     def __str__(self):
-        return f"trigger: {self.trigger} {self.value} loops togo: {self.togo}"
+        return f"trigger: {self.trigger_type} {self.value} loops togo: {self.togo}"
 
     @classmethod
     def fromConfig(cls, config=None):
         if not config:
```

Comment:
Ive moved python requirement to >3.10, so can use switch approach ...

Reply:
I am not sure what you mean. I just renamed this variable to help with readability.

Reply:
ahh, was a comment related to the if, elif block - just a note as before I rejected strenum due to python version but now Ive been convinced to bump minimum version....;